css/components/_main-sidebar.scss

```
@@ -1,6 +1,23 @@
 .main-sidebar {
-    padding: 40px 0;
-    transition: transform .3s linear;
+    padding: 40px 0 0;
+    transition: transform $transition-duration $transition-function;
```

Replace the usage everywhere.
```suggestion
    transition: transform $transition-base;
```